internal/env/config.go

```diff
@@ -21,35 +21,40 @@ var (
+	Namespace          string   ``                                          // Kubernetes namespace for all helm commands
+	KubeToken          string   `split_words:"true"`                        // Kubernetes authentication token to put in .kube/config
+	SkipTLSVerify      bool     `envconfig:"skip_tls_verify"`               // Put insecure-skip-tls-verify in .kube/config
+	Certificate        string   `envconfig:"kube_certificate"`              // The Kubernetes cluster CA's self-signed certificate (must be base64-encoded)
+	APIServer          string   `envconfig:"kube_api_server"`               // The Kubernetes cluster's API endpoint
+	ServiceAccount     string   `envconfig:"kube_service_account"`          // Account to use for connecting to the Kubernetes cluster
+	ChartVersion       string   `split_words:"true"`                        // Specific chart version to use in `helm upgrade`
+	DryRun             bool     `split_words:"true"`                        // Pass --dry-run to applicable helm commands
+	Wait               bool     `envconfig:"wait_for_upgrade"`              // Pass --wait to applicable helm commands
+	ReuseValues        bool     `split_words:"true"`                        // Pass --reuse-values to `helm upgrade`
+	KeepHistory        bool     `split_words:"true"`                        // Pass --keep-history to `helm uninstall`
+	Timeout            string   ``                                          // Argument to pass to --timeout in applicable helm commands
+	Chart              string   ``                                          // Chart argument to use in applicable helm commands
+	Release            string   ``                                          // Release argument to use in applicable helm commands
+	Force              bool     `envconfig:"force_upgrade"`                 // Pass --force to applicable helm commands
+	AtomicUpgrade      bool     `split_words:"true"`                        // Pass --atomic to `helm upgrade`
+	CleanupOnFail      bool     `envconfig:"cleanup_failed_upgrade"`        // Pass --cleanup-on-fail to `helm upgrade`
+	LintStrictly       bool     `split_words:"true"`                        // Pass --strict to `helm lint`
+	ConvertV2Releases  bool     `split_words:"true"`                        // Whether or not to use 2to3 convert to migrate Releases from v2 to v3
+	DeleteV2Releases   bool     `split_words:"true"`                        // Pass --delete-v2-releases option for 2to3 convert command
+	ReleaseVersionsMax int      `split_words:"true"`                        // Pass --release-versions-max option for 2to3 convert command
+	TillerNS           string   `split_words:"true" default:"kube-system"`  // Tiller namespace (--tiller-ns) for 2to3 convert command
+	TillerLabel        string   `split_words:"true" default:"OWNER=TILLER"` // Tiller label selector (--label) for 2to3 convert command
```

Comment:
**question:** I don't see where this is defaulting to `OWNER=TILLER`. This seems to be processed by a command flag in the 2to3 library, which is not being used here. What happens if this is ""?

Reply:
It's defined in the 2to3 library: https://github.com/helm/helm-2to3/blob/master/pkg/v2/release.go#L128

If the value is empty here, it will still default to `OWNER=TILLER`, and `kube-system` in TillerNS.

I also found an issue when using default values in envconfig that cannot be overriden later for some reason, so I decided to not set default values in envconfig and instead rely on default behaviour

Reply:
I removed the default values from the envconfig tags